RentACar/RentACar/Repositories/Repository.cs

```diff
@@ -0,0 +1,64 @@
+﻿using RentACar.Helpers;
+using RentACar.Intefaces;
+using RentACar.Models.Base;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RentACar.Repositories
+{
+    public abstract class Repository<T> : IRepository<T> where T : Entity
+    {
+        private List<T> _objs;
+
+        private readonly string _jsonFile;
+
+        public Repository(string config)
```

Comment:
same here.. does `config` represent what this parameter is use for?

Reply:
Not at all.. you are right, thanks!  maybe storagePath?

Reply:
IMHO that sounds a lot better